.conan/conan-build.sh

```diff
@@ -1,37 +1,3 @@
 #!/bin/bash
 set -eo pipefail
```

Comment:
Could we put a quick comment in the file to explain why the file looks incomplete? I think that'd help remove confusion for others.

Reply:
Done.